### PR TITLE
Removed deprecations for Image constants, except for duplicate Resampling attributes

### LINKED ISSUE #6537: API changes for Resampling modes
With Pillow 9.2.0, I get the following deprecation notice:

```
DeprecationWarning: BILINEAR is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BILINEAR instead.
```

Is it really necessary to introduce this breaking API change? Can the old names not be kept for backwards compatibility? Thousands of repositories will need to update their code, wasting countless man-hours.

You can get a rough idea on how many repositories are affected from the following two websites, which search a subset of GitHub repositories:

* https://grep.app/search?q=Image.BILINEAR
* https://sourcegraph.com/search?q=context:global+Image.BILINEAR&patternType=standard

```python
from PIL import Image
img = Image.new("RGB", (5, 5))
img.resize((7, 7), Image.BILINEAR)
```

### PR DESCRIPTION
**Pull request by radarhere**
Resolves #6537

The user in the issue found a warning that `Image.BILINEAR` is deprecated, and questions whether it is necessary to deprecate this.

This PR removes the deprecation warnings for `Image.*` constants added in #5954 ([and released in Pillow 9.1.0](https://pillow.readthedocs.io/en/stable/releasenotes/9.1.0.html#constants)).
However, there are three exceptions - I've left `LINEAR`, `CUBIC` (which have been duplicates of other constants since PIL) and `ANTIALIAS` (which has been a duplicate since #1028) deprecated.

I'm also leaving the deprecations for ImageCms, ImageFont, BlpImagePlugin, FtexImagePlugin and PngImagePlugin in place.

See what you think of this.

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for picking this up. Looks fine, let's keep this open a few days and merge before January if there's no strong objections.

Let's also mention it in the release notes, and add a note (with `.. note::` ?) to https://pillow--6830.org.readthedocs.build/en/6830/deprecations.html#constants to mention that some had been deprecated but are no longer.

Also, I wonder if we should somehow mention it at https://pillow--6830.org.readthedocs.build/en/6830/releasenotes/9.1.0.html#constants? These are historical, but perhaps some sort of note, even if just a link to the 9.3.0 notes, could help?

**Comment by radarhere** [comment]
Ok, I've added a commit to update the documentation.

File: docs/releasenotes/9.4.0.rst [hunk 1]
```diff
@@ -103,3 +103,40 @@ Added support for DDS L and LA images
...
 Support has been added to read and write L and LA DDS images in the uncompressed
 format, known as "luminance" textures.
+
+.. _restored-image-constants:
+
+Constants
+^^^^^^^^^
+
+In Pillow 9.1.0, the following constants were deprecated. Those deprecations have now
+been restored.
```
Comment (hugovk) [inline docs/releasenotes/9.4.0.rst:113]: Wonder if something like this is clearer?
```suggestion
In Pillow 9.1.0, the following constants were deprecated.
That has been reversed and these constants will now remain available.
```

Comment (radarhere) [reply docs/releasenotes/9.4.0.rst]: Ok, sure. I've pushed a commit changing that, and the text in deprecations.rst.

**Review by hugovk — APPROVED** [review]

**Pull request merged**